ts/utils/__tests__/isVersionAppSupported.test.ts

```diff
@@ -0,0 +1,25 @@
+import { ServerInfo } from "../../../definitions/backend/ServerInfo";
+import { isVersionAppSupported } from "../isVersionAppSupported";
+
+describe("isVersionAppSupported", () => {
+  it("is supported", () => {
+    const backendInfo: ServerInfo = {
+      version: "123",
+      minAppVersion: "0.0.0"
+    };
+    expect(isVersionAppSupported(backendInfo, "1.2")).toEqual(true);
+  });
+
+  it("not supported", () => {
+    const backendInfo: ServerInfo = {
+      version: "123",
+      minAppVersion: "1.4.0"
+    };
+    expect(isVersionAppSupported(backendInfo, "1.2")).toEqual(false);
+  });
+
+  it("undefined then supported", () => {
+    const backendInfo = undefined;
+    expect(isVersionAppSupported(backendInfo, "1.2")).toEqual(true);
+  });
+});
```

Comment:
If I run this test, it fails

```typescript
it("test", () => {
    const backendInfo: ServerInfo = {
      version: "123",
      minAppVersion: "1.4.5"
    };
    expect(isVersionAppSupported(backendInfo, "1.4.1")).toEqual(false);
  });
```

this is because using **parseFloat** you have this output:
```typescript
const test = "1.2.3";
parseFloat(test); // -> 1.2
```
so the last digit is never read

Furthermore the implementation of **isVersionAppSupported** doesn't support the versioning using 4 digits (we could use it in future)

You could try to use [this library](https://www.npmjs.com/package/semver) to make software version comparison